docs/content/packages/react.mdx

```diff
@@ -66,6 +66,23 @@ export default () => (
 )
 ```
 
+### `focusable`
+
+You can add the `focusable` attribute to the SVG element via the `focusable` prop, which can be either `true`, `false`, or `auto`.
```

Comment:
Is the intent with `focusable="true"` for the `<svg>` to be interactive/be able to receive focus? If so, how would you feel about the prop being exposed here being `tabIndex` and that would be the signal for `focusable` being `true` or `false`?

In other words:

```tsx
<Icon />                                // <svg aria-hidden="true" focusable="false">
<Icon aria-label="test" />              // <svg aria-label="test" role="img" focusable="false">
<Icon aria-label="test" tabIndex={0} /> // <svg aria-label="test" role="img" tabindex="0" focusable="true">
```

If I understand right, `focusable` is only implemented in IE11 and older versions of Edge and so may not function as expected in all browsers if the intent is for the svg to be focusable in that situation.

Most of this is just based on: https://allyjs.io/tutorials/focusing-in-svg.html. Let me know if I'm misunderstanding anything!